### PR TITLE
Explain a downside of running multiple schedulers

### PR DESCRIPTION
**Pull request by diversario**
/kind documentation

Provide concrete example of how running multiple schedulers
can lead to scheduling conflicts.

### CODE REVIEW

File: doc/install.md [hunk 1]
```diff
@@ -27,14 +27,18 @@ If you do not have a cluster yet, create one by using one of the following provi
...
-the privileges to manipulate on control plane, also for this moment, the installation is not fully
-automated (no Helm chart yet).
+- **second scheduler:**
+  - **pro**: it's easy to install by deploying the Helm chart
+  - **con**: running multi-scheduler will inevitably encounter resource conflicts when the cluster is short of resources.
+
+    Consider the scenario where multiple schedulers attempt to assign their pods simultaneously to a node which can only fit one of the pods.
+    The pod that arrives later will be evicted by the kubelet, and hang there (without its `.spec.nodeName` cleared) until resources get released on the node.
+
+    Running multiple schedulers, therefore, is not recommended in the production env. However, it's a good starting point to play with.
```
Comment (Huang-Wei) [inline doc/install.md:37]: nit: remove the ending `.`

```suggestion
    Running multiple schedulers, therefore, is not recommended in the production env. However, it's a good starting point to play with
```

**Comment by diversario** [comment]
Not sure what the `do-not-merge/release-note-label-needed` label actually wants me to do. I read the message but seems like it shouldn't apply for this change?

**Pull request merged**